### PR TITLE
New Sniff Styles Bonanza

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

File: tests/Behance/Sniffs/Operators/OperatorSpacingSniffTest.inc.fixed [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+$truthy = 1 > 2;
+$truthy = 1 > 2;
+$truthy = 1  >  2;
+$truthy = 1
+              >
+                   2;
+
+$truthy = 1 >= 2;
+$truthy = 1 >= 2;
+$truthy = 1  >=  2;
```
Comment (mikesherov) [inline tests/Behance/Sniffs/Operators/OperatorSpacingSniffTest.inc.fixed]: Hmmm. Still need to fix this!

File: Behance/ruleset.xml [hunk 2]
```diff
@@ -77,32 +60,25 @@
...
     </properties>
   </rule>
+
+  <!-- Custom standards -->
+  <rule ref="Behance.Arrays.ArrayBracketSpacing"/>
+  <rule ref="Behance.Arrays.ArrayDeclaration"/>
   <rule ref="Behance.Arrays.ShortHandArrayOnly"/>
   <rule ref="Behance.Classes.ClassDeclaration"/>
   <rule ref="Behance.Comments.DisallowHashComments"/>
   <rule ref="Behance.Comments.TrailingComment"/>
```
Comment (remotelyliving) [inline Behance/ruleset.xml]: Did we want to kill this one?

File: tests/Behance/Sniffs/Arrays/ArrayDeclarationSniffTest.inc [hunk 3]
```diff
@@ -1,134 +1,141 @@
...
+  'testitup' => array('something', 'a thing'),
+];
 
+$uhohBadArrowsComingUp = [
+  'test'     => 1,
+  'testitup' => 2,
+  'tests' =>    3,
+  'come'  =>    4,
+  'on'  =>      5,
+  'fqhgwadssas'  => 6,
```
Comment (amazd) [inline tests/Behance/Sniffs/Arrays/ArrayDeclarationSniffTest.inc]: :D

File: Behance/ruleset.xml (hunk 2 above)
Comment (mikesherov) [reply Behance/ruleset.xml]: I’m reversing it.

**Pull request merged**